src/resources/pandoc/datadir/_json.lua

```diff
@@ -26,7 +27,22 @@
 -- SOFTWARE.
 --
 
-local json = { _version = "0.1.2" }
+local json = { _version = "0.1.2-quarto" }
```

Comment:
we can't keep the version number but I'm not sure if this is a good way to deal with it

Reply:
This is fine.